docs/markdown/recursor/scripting.md

```diff
@@ -99,6 +99,7 @@ The DNSQuestion object contains at least the following fields:
     * policyCustom: The CNAME content for the `pdns.policyactions.Custom` response, a string
     * policyTTL: The TTL in seconds for the `pdns.policyactions.Custom` response
 * wantsRPZ - A boolean that indicates the use of the Policy Engine, can be set to `false` in `preresolve` to disable RPZ for this query
+* data - a table that is persistent throughout the lifetime of the `dq` object and can be used to store custom context.
```

Comment:
Yes, I understand how the data, in your context (haha), is context. I think 'data' is fine. Maybe also note that it is strictly string/string, you can't put tables inside this table like you can with normal Lua tables.